docs/designers-developers/developers/data/data-core.md

```diff
@@ -471,6 +471,17 @@ _Returns_
 
 -   `Object`: Action object.
 
+<a name="deleteEntityRecord" href="#deleteEntityRecord">#</a> **deleteEntityRecord**
+
+Action triggered to delete an entity record.
+
+_Parameters_
+
+-   _kind_ `string`: Kind of the deleted entity.
+-   _name_ `string`: Name of the deleted entity.
+-   _recordId_ `string`: Record Id of the entity to be deleted.
+-   _deleteQueryParams_ `string`: Special query parameters for the DELETE API call. These parameters will be appended to the rest base with a '?' in front.
```

Comment:
This should be `query` to match the other functions.

Reply:
done!